pages/documentation/developer/tutorials/emf.md

```diff
@@ -0,0 +1,361 @@
+### Create parameters to add options to the calculation
+Then we define three parameters: a boolean indicating whether or not we want to perform the load flow pre-processing on the IGMs, a boolean indicating whether we want to prepare the balance computation or not and the name of the synchronous area, set to `10YEU-CONT-SYNC0`, representing Continental Europe.
+
+### Import the CIM-CGMES IGMs
+Now we move to the main method.
+First, we create the `BalancesAdjustmentValidationParameters` variable, from the class you have created before, to read the paths to the IGMs, the PEVF file and the output directory. You can now start logging into the output directory by creating a specific `log` method taking the `validationParameters` as an argument and checking if the output directory exists or not to set it as the output path.
+
+```java
+BalancesAdjustmentValidationParameters validationParameters = BalancesAdjustmentValidationParameters.load();
+log(validationParameters);
+```
+Then we create a special method to import the IGMs outside the Main method. In this method, a HashMap of networks is created and for each IGM path in the validation parameters, the corresponding IGM is loaded through `Importers`. We call this method in Main to import the networks with:
+```java
+Map<String, Network> networks = importNetworks(validationParameters);
+```
+
+### Power flow on the IGMs
+Then we compute a power flow on the networks in order to have an idea tof he valid ones, those for which the load flow is successful. In case of non-convergence of the load flow on an IGM, it is preferable to relax the parameters (tap changer and shunt regulations can be switched off, the reactive power limits of generators can be violated). In our tutorial, we have decided to remove the IGM but it should be the final solution. We create a new method `loadflowPreProcessing`, that runs the load flow via OpenLoadFlow and we add the corresponding code in the main method:
+```java
+Map<String, Network> validNetworks = new HashMap<>(networks);
+if (LOAD_FLOW_PREPROCESSING) {
+    loadflowPreProcessing(networks, validNetworks);
+}
+```
+
+### Merge of the IGMs and power flow on the CGM
+Finally, we merge the IGMs to create the CGM:
+```java
+MergingView mergingView = MergingView.create("merged", "validation"); 
+mergingView.merge(validNetworks.values().toArray(Network[]::new));
+```
+And run a load flow on the CGM. Note that the slack bus is not defined in the CGMES files for the whole CGM, so we prefer to select the most meshed one.
+```java
+if (!PREPARE_BALANCE_COMPUTATION) {
+    LOAD_FLOW_PARAMETERS.setReadSlackBus(false);
+    LOAD_FLOW_PARAMETERS.setDistributedSlack(true);
+    LoadFlowResult result = LoadFlow.run(mergingView, LOAD_FLOW_PARAMETERS);
+    for (Generator gen : mergingView.getGenerators()) {
+        gen.setTargetP(-gen.getTerminal().getP());
+    }
+    System.out.println(result.isOk());
+    System.out.println(result.getMetrics());
+}
+```
+
+### Balance computation
```

Comment:
Is there a rendering issue here? The title appears different (it might be a github issue, if the rendering is okay on your computer, ignore this remark).

Reply:
It looks ok on my side